insteon_mqtt/device/KeypadLinc.py

```diff
@@ -870,6 +870,36 @@ def set_on_level(self, level, on_done=None):
 
         self.send(msg, msg_handler)
 
+    #-----------------------------------------------------------------------
+    def set_ramp_rate(self, rate, on_done=None):
+        """Set the device default ramp rate.
+
+        This changes the dimmer default ramp rate of how quickly the it
+        will turn on or off. This rate can be between .1 seconds and up
+        to 9 minutes.
+
+        Args:
+          rate (int): The default ramp rate in the range [0,31]
+          on_done: Finished callback.  This is called when the command has
+                   completed.  Signature is: on_done(success, msg, data)
+        """
+        LOG.info("Dimmer %s setting ramp rate to %s", self.label, rate)
```

Comment:
I suspect nothing bad would happen.  But it would be nice to check if this keypadlinc is dimmer capable.  Otherwise I don't think the ramp_rate is necessary or would work.

Something like would do it.
```python
if !self.is_dimmer:
  LOG.error("Can't set ramp rate device %s is not a dimmer", self.label)
  return
```

Reply:
Good call! I added the check for is_dimmer like you suggested. I forgot some keypads aren't dimmers.